Producer/Producer.Droid/Framework/Fragments/SettingsFragment.cs

```diff
@@ -0,0 +1,85 @@
+﻿using Android.Views;
+using Android.OS;
+using Android.Content;
+using Producer.Domain;
+using Android.Preferences;
+using Android.Support.V4.Content;
+using SettingsStudio;
+using System.Collections.Generic;
+
+namespace Producer.Droid
+{
+	public class SettingsFragment : PreferenceFragment
+	{
+		static readonly string [] Preferences = { 
+			SettingsKeys.TestProducer,
+			SettingsKeys.MobileCenterKey,
+			SettingsKeys.RemoteFunctionsUrl,
+			SettingsKeys.RemoteDocumentDbUrl,
+			SettingsKeys.NotificationsName,
+			SettingsKeys.NotificationsConnectionString,
+			SettingsKeys.EmbeddedSocialKey,
+			SettingsKeys.UserReferenceKey
+		};
+
+		public override void OnCreate (Bundle savedInstanceState)
+		{
+			base.OnCreate (savedInstanceState);
+			SetHasOptionsMenu (true);
+			AddPreferencesFromResource (Resource.Xml.preferences);
+			FindPreference ("VersionNumberString").Summary = $"{Settings.VersionNumber} ({Settings.BuildNumber})";
+		}
+
+		public override View OnCreateView (LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState)
+		{
+			var view = base.OnCreateView (inflater, container, savedInstanceState);
+			var color = new Android.Graphics.Color (ContextCompat.GetColor (this.Context, Resource.Color.primary));
+			view.SetBackgroundColor (color);
+			return view;
+		}
+
+		public override void OnCreateOptionsMenu (IMenu menu, MenuInflater inflater)
```

Comment:
Is this override necessary?  If not, remove it, else maybe a few words in a comment saying don't delete it.

Reply:
@colbylwilliams in this case this override is necessary since we are customizing the toolbar with a menu xml to add the settings button. regards!